.buildkite/pipeline.yml

```diff
@@ -42,12 +42,12 @@ steps:
       #   artifact_paths: "test/*"
 
       - label: "Moist earth with slab surface"
-        command: "julia --color=yes --project=experiments/AMIP/moist_mpi_earth/ experiments/AMIP/moist_mpi_earth/coupler_driver.jl --energy_check=true --mode_name=slabplanet --t_end=10days --dt_save_to_sol=3600secs --dt_cpl=200 --mono_surface=true"
+        command: "julia --color=yes --project=experiments/AMIP/moist_mpi_earth/ experiments/AMIP/moist_mpi_earth/coupler_driver.jl --energy_check=true --mode_name=slabplanet --t_end=10days --dt_save_to_sol=3600secs --dt_cpl=200 --mono_surface=true --h_elem=4"
```

Comment:
Is there a reason (apart from because it is cheaper and faster) why we run such a low resolution setup in ci? Since our model seems very sensitive to any tiny changes in the setup, I am concerned that successful simulation in this setup won't guarantee any successful runs with the targeted setup

Reply:
That's right! I'm actually setting up a PR with the target resolution as we speak (with MPI and ClimaTImesteppers), which will be a more accurate benchmark. :)